scripts/build_windows.bat

```diff
@@ -61,7 +61,7 @@ cd %CAFFE2_ROOT%\build
 
 :: Set up cmake. We will skip building the test files right now.
 :: TODO: enable cuda support.
-cmake .. ^
+cmake ^
```

Comment:
So long as we don't use build_windows.bat in any of our CI testing I'd recommend removing it.

Reply:
It seems that it is used in the CI `pr/caffe2-py2-cuda9.0-cudnn7-windows-build`.